dir.targets

```diff
@@ -52,15 +57,21 @@
 
     <!-- Restore build tools -->
     <Exec Command="$(NugetRestoreCommand) &quot;$(SourceDir).nuget\packages.config&quot;" StandardOutputImportance="Low" />
-    
-    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true'"
-           Text="The build tools package was just restored and so we cannot continue the build of an individual project because targets from the build tools package were not able to be imported. Please retry the build the individual project again." />
+
+    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true' and '$(BuildingInsideVisualStudio)'=='true'"
+      Text="The build can not be completed as the build tools package was just restored. Please reload the solution to allow targets from the build tools package to be imported." />
```

Comment:
Your right that kind of sucks though. I wonder it isn't easier for us to tell people to just close VS and reopen it to fix this issue.
/cc @nguerrera @terrajobst thoughts on this?